teps/README.md

```diff
@@ -140,10 +140,17 @@ thus a feature) in the tektoncd projects.
    [the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api)
    at the latest.
 
-Reviewers should use [`/approve`](../process.md#prow-commands) to indicate that they approve of the PR being merged. Once all assigned reviewers
-have approved the PR, the final [`/lgtm`](../process.md#prow-commands) can be added in
-[the next API working group meeting](https://github.com/tektoncd/community/blob/main/working-groups.md#api) or sooner
-by anyone with permission to if needed. If a reviewer has not submitted an `/approve`, this is taken to mean that the
+Reviewers should use [`/approve`](../process.md#prow-commands) to indicate that they approve of the PR being merged.
+Once all assigned reviewers have approved the PR, anyone with permissions
+may merge the PR by adding a [`/lgtm` label](../process.md#prow-commands), as long as the author
+has addressed any feedback from reviewers and the TEP has not undergone substantial changes
+since any reviewers have approved it. If the TEP has undergone substantial changes since a
+reviewer approved it, the author should communicate these changes to the reviewer, and either
+person should comment on the PR stating that the reviewer is OK with the design changes before
+the PR is merged. (Note: automation prevents the PR author from merging their own PR.)
+TEP PRs do not need to be merged during any synchronous meeting.
```

Comment:
Should we still continue to review TEPs in the [API WG](https://docs.google.com/document/d/17PodAxG8hV351fBhSu7Y_OIPhGTVgj6OJ2lPphYYRpU/edit#heading=h.3zh9uuzdo77w) since its no longer part of the guidelines?